src/poliastro/twobody/angles.py

```diff
@@ -172,7 +172,7 @@ def fp_angle(nu, ecc):
     ecc : float
         Eccentricity.
 
-    Notes
+    Note
```

Comment:
We use "Notes" to be consistent with https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt